src/FffPolygonGenerator.cpp

```diff
@@ -31,6 +32,17 @@ namespace cura
 
 bool FffPolygonGenerator::generateAreas(SliceDataStorage& storage, MeshGroup* meshgroup, TimeKeeper& timeKeeper)
 {
+#pragma omp parallel
+    {
+#pragma omp master
+        {
+#ifdef _OPENMP
+            log("OpenMP enabled, number of threads used: %u\n", omp_get_num_threads());
+#else
+            log("OpenMP disabled\n");
+#endif
+        }
+    }
```

Comment:
Why is this logging in the `generateAreas` function? Wouldn't you want this in a (very) central location, such as Main or FffProcessor?